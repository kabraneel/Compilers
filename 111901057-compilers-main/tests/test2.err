IR PRETTY PRINT
.globl main
main:
li t1, 2
li t3, 3
li t4, 4
add t2, t3, t4
li t8, 10
syscall

 BASIC BLOCKS 
.globl main

main:
li $t1, 2
li $t3, 3
li $t4, 4
add $t2, $t3, $t4
li $v0, 10
syscall

