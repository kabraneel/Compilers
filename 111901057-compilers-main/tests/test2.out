.globl main
main:
li $t1, 2
li $t3, 3
li $t4, 4
add $t2, $t3, $t4
li $v0, 10
syscall
